.eslintrc.js

```diff
@@ -0,0 +1,97 @@
+module.exports = {
```

Comment:
Why use `.eslintrc.js` instead of a regular `.eslintrc`? What are the benefits out of curiosity?